src/DataGrid.tsx

```diff
@@ -871,7 +848,7 @@ function DataGrid<R, SR>({
           onRowClick={onRowClick}
           rowClass={rowClass}
           top={top}
-          copiedCellIdx={copiedPosition?.rowIdx === rowIdx ? copiedPosition.idx : undefined}
+          copiedCellIdx={copiedCell?.row === row ? copiedCell.column.idx : undefined}
```

Comment:
Do we really need to style copied cells?

Reply:
I think so, excel also adds styles

![image](https://user-images.githubusercontent.com/1452717/97734523-47b77c80-1aa7-11eb-87af-2473b84ac0c2.png)